stroeer/core/v1/article.proto

```diff
@@ -689,6 +692,7 @@ message Article {
      * | `DISCLAIMER`        | A article disclaimer with important notes/legal stuff. E.g. "medizinischer Hinweis" on all medical articles |
      * | `TABLE_OF_CONTENTS` | Table of contents for this article, consists of anchors which refer to sub headlines within the `BODY`      |
      * | `ARTICLE_EXTENDER`  | AI generated questions for the article for more engagement.                                                 |
+     * | `COMPANION`         | Rich text to be rendered as teaser within a stage companion                                                 |
```

Comment:
Pls no. Just send us two teasers:

```
<stage>
  <stream>
    <Teaser body="BODY" />
  </stream>
  <companions>
    <Teaser body="BODY" />
  </companions>
</stage>
```

Reply:
As viewed by the Adapter // Core-API: 

It's not two distinct items. It's one item that has two parts. 

- first part goes into the content column (teaser text)
- second part goes into the companion column (image + table)

Paperboy can model it this way when creating the `SectionPageResponse`. But this is not what's happening in the CMS/Adapter.